docs/hub/datasets-manual-configuration.md

```diff
@@ -0,0 +1,142 @@
+# Manual Configuration
+
+To host and share your dataset, create a dataset repository on the Hugging Face Hub and upload your data files.
+
+This guide will show you how to configure a custom structure for your dataset repository.
+A dataset with a supported structure and file format (`.txt`, `.csv`, `.parquet`, `.jsonl`, `.mp3`, `.jpg`, `.zip` etc.) automatically has a dataset viewer on its dataset page on the Hub.
+
+## Define your splits and subsets in YAML
+
+## Splits
+
+If you have multiple files and want to define which file goes into which split, you can use the YAML `configs` field at the top of your README.md.
```

Comment:
maybe explain (or link to) what a config is?

Reply:
it might be confusing with the title of this page (Manual configuration)

Reply:
I removed the mention of `configs` in this sentence. And added this in the intro of this page to introduce the notion of configs:

```
You can use YAML to configure the splits and builder parameters that are used by the Viewer.

It is even possible to define multiple configurations for the same dataset (e.g. if the dataset has various independent files).
```